src/psyclone/psyir/symbols/__init__.py

```diff
@@ -33,32 +33,35 @@
 # -----------------------------------------------------------------------------
 # Author S. Siso, STFC Daresbury Lab
 # Modified by A. R. Porter and R. W. Ford, STFC Daresbury Lab
+# Modified by J. Henrichs, Bureau of Meteorology
 # -----------------------------------------------------------------------------
 
 ''' Symbols package module '''
 
 from psyclone.psyir.symbols.datasymbol import DataSymbol
 from psyclone.psyir.symbols.containersymbol import ContainerSymbol
 from psyclone.psyir.symbols.data_type_symbol import DataTypeSymbol
-from psyclone.psyir.symbols.interfaces import AutomaticInterface, \
-    ImportInterface, ArgumentInterface, UnresolvedInterface, StaticInterface, \
-    DefaultModuleInterface, UnknownInterface, CommonBlockInterface
+from psyclone.psyir.symbols.interfaces import (
+     ArgumentInterface, AutomaticInterface, CommonBlockInterface,
+     DefaultModuleInterface, ImportInterface,
+     StaticInterface, UnknownInterface, UnresolvedInterface)
 from psyclone.psyir.symbols.intrinsic_symbol import IntrinsicSymbol
```

Comment:
Not your fault but I noticed that IntrinsicSymbol is missing from the `__all__` list

Reply:
Done.